ocaml/testsuite/tests/parsetree/locations_test.ml

```diff
@@ -132,3 +132,7 @@ let g y =
   let f ~y = y + 1 in
   f ~(y:int)
 ;;
+
+(* Newtype *)
+let f (type a b c d e) = ()
+;;
```

Comment:
Could you add a test using the `fun` syntax as well?

Reply:
test case added